eq-publisher/src/eq_schema/Section.test.js

```diff
@@ -138,11 +120,5 @@ describe("Section", () => {
       expect(introBlock.type).toBe("Interstitial");
       expect(introBlock.description).toBe(description);
     });
-
```

Comment:
This test didn't have to come out. It relates to the intro block title not the section/group title.

Reply:
Doh!